docs/index.rst

```diff
@@ -0,0 +1,122 @@
+.. Images
+.. |rate_1| image:: images/rate_1.jpg
+.. |rate_2| image:: images/rate_2.jpg
+.. |rate_3| image:: images/rate_3.jpg
+
+.. Links
+.. |exprtk| raw:: html
+
+   <a href="http://www.partow.net/programming/exprtk/index.html">ExprTk</a>
+
+.. |fledge-south-expression| raw:: html
+
+   <a href="../../fledge-south-expression/index.html">fledge-south-expression</a>
+
+.. |fledge-filter-threshold| raw:: html
+
+   <a href="../fledge-filter-threshold/index.html">fledge-filter-threshold</a>
+
+
+.. |fledge-filter-expression| raw:: html
+
+   <a href="../fledge-filter-expression/index.html">fledge-filter-expression</a>
+
+Rate Control Filter
+===================
+
+A filter plugin that can be used to reduce the rate a reading is stored
```

Comment:
A `fledge-filter-rate` plugin is a filter that ...

Reply:
I got bored using the same words!